### PR TITLE
[Don't merge it before releasing OBS 2.7] Update links pointing to version 2.6

### PR DESCRIPTION
**Pull request by mdeniz**

### CODE REVIEW

**Comment by mdeniz** [comment]
After releasing OBS 2.7 maybe is needed an update of the books again.

**Comment by mdeniz** [comment]
@hennevogel @adrianschroeter please review :smile:

File: download/other/index.html [hunk 1]
```diff
@@ -101,16 +101,16 @@
...
     <h5>SUSE Linux Enterprise</h5>
     <pre>
-      zypper ar -f http://download.opensuse.org/repositories/OBS:/Server:/2.6/SLE_11_SP3/OBS:Server:2.6.repo
+      zypper ar -f http://download.opensuse.org/repositories/OBS:/Server:/2.7/SLE_11_SP3/OBS:Server:2.7.repo
       zypper in -t pattern OBS_Server
     </pre>
     <h5>openSUSE</h5>
     <pre>
-      zypper ar -f http://download.opensuse.org/repositories/OBS:/Server:/2.6/openSUSE_13.1/OBS:Server:2.6.repo
+      zypper ar -f http://download.opensuse.org/repositories/OBS:/Server:/2.7/openSUSE_13.1/OBS:Server:2.7.repo
```
Comment (adrianschroeter) [inline download/other/index.html]: 13.1 is no more ... better point to openSUSE_42.1

Comment (mdeniz) [inline download/other/index.html]: Ok, I will

File: download/other/index.html [hunk 2]
```diff
@@ -77,7 +77,7 @@
     </a>
   </div>
```
Comment (adrianschroeter) [inline download/other/index.html]: copy the block and point also to .vdi images for VirtualBox (the vmdk ones are not working anymore)

File: download/other/index.html [hunk 3]
```diff
@@ -101,16 +101,16 @@
     <p>
       If you already have a Linux server running you can use our binary packages to install OBS and follow our
       <a href="https://github.com/openSUSE/open-build-service/blob/master/README.md">README</a> to set everything
-      up yourself. Currently we maintain packages for the openSUSE and SUSE Linux Enterprise distributions. 
+      up yourself. Currently we maintain packages for the openSUSE and SUSE Linux Enterprise distributions.
     </p>
     <h5>SUSE Linux Enterprise</h5>
     <pre>
-      zypper ar -f http://download.opensuse.org/repositories/OBS:/Server:/2.6/SLE_11_SP3/OBS:Server:2.6.repo
+      zypper ar -f http://download.opensuse.org/repositories/OBS:/Server:/2.7/SLE_11_SP3/OBS:Server:2.7.repo
```
Comment (hennevogel) [inline download/other/index.html]: Please point to the SLE_12 repo

File: download/other/index.html (hunk 2 above)
Comment (hennevogel) [inline download/other/index.html]: I'll do this afterward

**Comment by hennevogel** [comment]
LGTM

**Pull request merged**